js/authJs/loginForm.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const favicon = document.getElementById("favicon"); // icon
+  const passwordInput = document.getElementById("password"); // 비밀번호
+  const emailInput = document.getElementById("email"); // 이메일
+  const emailError = document.getElementById("emailError"); // 이메일 에러구문
+  const passwordError = document.getElementById("passwordError"); // 비밀번호 에러구문
+  const loginButton = document.getElementById("loginButton"); // 로그인 버튼
+
+  // 이메일 포커스 아웃 시 유효성 검사
+  emailInput.addEventListener('blur', function () {
+    if (!emailInput.value) {
+      emailInput.classList.add('error');
+      emailInput.classList.remove("valid");
+      emailError.textContent = "이메일을 입력해주세요";
+    } else if (!validateEmail(emailInput.value)) {
+      emailInput.classList.add("error");
+      emailInput.classList.remove("valid");
+      emailError.textContent = "잘못된 이메일 형식입니다";
+    } else {
+      emailInput.classList.remove("error");
+      emailInput.classList.add("valid");
+      emailError.textContent = "";
+    }
+    checkFormValidity();
+  });
+
+  // 비밀번호 포커스 아웃 시 유효성 검사
+  passwordInput.addEventListener("blur", function () {
+    if (!passwordInput.value) {
+      passwordInput.classList.add("error");
+      passwordInput.classList.remove("valid");
+      passwordError.textContent = "비밀번호를 입력해주세요";
+    } else if (passwordInput.value.length < 8) {
```

Comment:
비밀번호가 지금은 8글자까지인데 이후에 바뀔 수 있고, 단순한 숫자값이기에 상수를 정의해서 써주시면 더 좋을 거 같아요!

```suggestion
  const MINIMUM_PW_LENGTH = 8
 
 ...

    } else if (passwordInput.value.length < MINIMUM_PW_LENGTH) {
```